apps/finance/app/package.json

```diff
@@ -4,22 +4,22 @@
   "private": true,
   "license": "AGPL-3.0-or-later",
   "dependencies": {
-    "@aragon/api": "^1.0.0-beta.1",
+    "@aragon/api": "^1.0.0",
     "@aragon/templates-tokens": "^1.1.1",
-    "@aragon/ui": "^0.31.0",
+    "@aragon/ui": "^0.33.0",
     "@babel/polyfill": "^7.0.0",
     "bn.js": "^4.11.8",
     "date-fns": "2.0.0-alpha.22",
     "lodash.throttle": "^4.1.1",
     "prop-types": "^15.6.0",
     "qrcode.react": "^0.8.0",
-    "react": "^16.2.0",
+    "react": "^16.8.4",
     "react-display-name": "^0.2.3",
-    "react-dom": "^16.2.0",
+    "react-dom": "^16.8.4",
     "react-spring": "^7.2.8",
-    "rxjs": "^6.2.1",
+    "rxjs": "^6.4.0",
     "seed-random": "^2.2.0",
-    "styled-components": "4.1.3",
+    "styled-components": "^4.1.3",
```

Comment:
I noticed the other apps also have this pinned, and I think when I asked @bpierre he said those could also be unpinned? Should we make this unpinned for all the apps?

Reply:
Hmm, I'll remove it from this PR and maybe we can take care of it in a different one.